library/node

```diff
@@ -1,34 +1,43 @@
 # maintainer: Joyent Image Team <image-team@joyent.com> (@joyent)
 
-0.10.36: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0.10: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-0: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
-latest: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10
+0.10.36: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
+0.10: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10
 
 0.10.36-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 0.10-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-0-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
-onbuild: git://github.com/joyent/docker-node@d23f190e500e91ecc636878a079ff971b29eab3e 0.10/onbuild
 
-0.10.36-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0.10-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-0-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.10/slim
+0.10.36-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
+0.10-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.10/slim
 
-0.11.16: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
-0.11: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11
+0.11.16: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
+0.11: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11
 
 0.11.16-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 0.11-onbuild: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/onbuild
 
-0.11.16-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
-0.11-slim: git://github.com/joyent/docker-node@ebfa426f39062608dbfacabc656095cd08716c9c 0.11/slim
+0.11.16-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
+0.11-slim: git://github.com/joyent/docker-node@a1aefc91ac380239998a9f8521bf233ee524d8d0 0.11/slim
```

Comment:
Removing the tag here doesn't remove the image from the Hub.  For an example of this in practice, see https://registry.hub.docker.com/_/redis/ (just `2.6.17` and `2.8.19`) vs https://registry.hub.docker.com/_/redis/tags/manage/ (lots of old legacy versions in the `2.8` series).

This list is the list of officially supported (and thus actively rebuilt) tags.  If there's a possibility that there _might_ be a 0.11.17 release, I'm OK with keeping this, but otherwise I think we'd be better off removing it here so it's not listed as officially supported (and the image will still be available for folks that need it).